Cargo.toml

```diff
@@ -262,6 +262,9 @@ gimli = { opt-level = 3 }
 miniz_oxide = { opt-level = 3 }
 object = { opt-level = 3 }
 rustc-demangle = { opt-level = 3 }
+timely = { opt-level = 3 }
+mz-compute = { opt-level = 3 }
+mz-transform = { opt-level = 3 }
```

Comment:
How much do these slow down debug compile times?

Reply:
Oh, I've meant to remove them! Sorry for the noise! (Don't seem to negatively impact local build times)